fpm.gemspec

```diff
@@ -47,9 +47,15 @@ Gem::Specification.new do |spec|
   spec.add_dependency("childprocess", "< 1.0.0") # license: ???
 
   # For calling functions in dynamic libraries
-  # This is pinned because 1.13.0 requires Ruby 2.3 or later
-  # Ref: https://github.com/jordansissel/fpm/issues/1708
-  spec.add_dependency("ffi", "~> 1.12.0") # license: GPL3/LGPL3
+  if (Gem::Platform.local.os == "darwin" &&
+      Gem::Platform.local.cpu == "arm64")
```

Comment:
At least on my M1 machine `Gem::Platform.local.cpu` returns `universal` instead of `arm64` 🤷